desktop/apps/auction/routes.js

```diff
@@ -37,22 +37,14 @@ export async function index (req, res, next) {
   const auctionArticles = new Articles(articles)
 
   try {
-    const html = makeTemplate('index.jade', {
-      basePath: res.app.get('views'),
-      locals: {
-        ...res.locals,
```

Comment:
Maybe, but it definitely shouldn't reuse locals since that data is designed to change